File: WebSite/GFast/common/models/Favoritos.php
<?php

namespace common\models;

use Yii;

/**
 * This is the model class for table "favoritos".
 *
 * @property int $fav_id
 * @property int $fav_idguitarras
 * @property int $fav_iduser
 * @property int|null $fav_idsaldo
 *
 * @property Guitarras $favIdguitarras
 * @property User $favIduser
 */
class Favoritos extends \yii\db\ActiveRecord
{
    /**
     * {@inheritdoc}
     */
    public static function tableName()
    {
        return 'favoritos';
    }

    /**
     * {@inheritdoc}
     */
    public function rules()
    {
        return [
            [['fav_idguitarras', 'fav_iduser'], 'required'],
            [['fav_idguitarras', 'fav_iduser', 'fav_idsaldo'], 'integer'],
            [['fav_idguitarras'], 'exist', 'skipOnError' => true, 'targetClass' => Guitarras::className(), 'targetAttribute' => ['fav_idguitarras' => 'gui_id']],
            [['fav_iduser'], 'exist', 'skipOnError' => true, 'targetClass' => User::className(), 'targetAttribute' => ['fav_iduser' => 'id']],
        ];
    }

    /**
     * {@inheritdoc}
     */
    public function attributeLabels()
    {
        return [
            'fav_id' => 'Fav ID',
            'fav_idguitarras' => 'Fav Idguitarras',
            'fav_iduser' => 'Fav Iduser',
            'fav_idsaldo' => 'Fav Idsaldo',
        ];
    }

    /**
     * Gets query for [[FavIdguitarras]].
     *
     * @return \yii\db\ActiveQuery
     */
    public function getFavIdguitarras()
    {
        return $this->hasOne(Guitarras::className(), ['gui_id' => 'fav_idguitarras']);
    }

    /**
     * Gets query for [[FavIduser]].
     *
     * @return \yii\db\ActiveQuery
     */
    public function getFavIduser()
    {
        return $this->hasOne(User::className(), ['id' => 'fav_iduser']);
    }
}

File: Android/app/src/main/java/com/example/gfastandroid/modelo/Favoritos.java
package com.example.gfastandroid.modelo;

public class Favoritos {


    private int fav_id, fav_idguitarras, fav_iduser;

    public Favoritos(int fav_id, int fav_idguitarras, int fav_iduser) {
        this.fav_id = fav_id;
        this.fav_idguitarras = fav_idguitarras;
        this.fav_iduser = fav_iduser;
    }

    public int getFav_id() {
        return fav_id;
    }

    public void setFav_id(int fav_id) {
        this.fav_id = fav_id;
    }

    public int getFav_idguitarras() {
        return fav_idguitarras;
    }

    public void setFav_idguitarras(int fav_idguitarras) {
        this.fav_idguitarras = fav_idguitarras;
    }

    public int getFav_iduser() {
        return fav_iduser;
    }

    public void setFav_iduser(int fav_iduser) {
        this.fav_iduser = fav_iduser;
    }
}
